62_CAD/common.hlsl

```diff
@@ -197,6 +211,8 @@ NBL_CONSTEXPR uint32_t InvalidTextureIdx = nbl::hlsl::numeric_limits<uint32_t>::
 NBL_CONSTEXPR MajorAxis SelectedMajorAxis = MajorAxis::MAJOR_Y;
 // TODO: get automatic version working on HLSL
 NBL_CONSTEXPR MajorAxis SelectedMinorAxis = MajorAxis::MAJOR_X; //(MajorAxis) (1 - (uint32_t) SelectedMajorAxis);
+NBL_CONSTEXPR float MsdfPixelRange = 4.0;
+NBL_CONSTEXPR float MsdfSize = 32.0; // TODO: Should this be dynamic?
```

Comment:
nope, but make sure to use MSDF all caps + use that on the c++ side as well

Reply:
Done